coalib/settings/FunctionMetadata.py

```diff
@@ -102,7 +102,7 @@ def _get_param(param, section, annotation):
 
         try:
             return annotation(section[param])
-        except:
+        except TypeError:
```

Comment:
@Uran198 you are right,  i mean't we should make the corresponding changes to `coala-bears`, if that is the case